### ISSUE TITLE
Exportable Settings.

### ISSUE THREAD

**Issue by Micasddsa4095** [johnnesky/beepbox#20]
The ability to export settings in a file, because I use a Chromebook when I'm using beepbox, when I restart my Chromebook, it clears my settings, a feature where I don't have to re-enable/disable the settings again would be nice.

**Comment by johnnesky** [comment]
Huh, that's weird, I don't think it's normal for Chromebook to reset Chrome settings on restart. Is it a school laptop? Are you using a guest account?

**Comment by Micasddsa4095** [comment]
Yes, it's a school laptop.

**Comment by Micasddsa** [comment]
Coming back to this, i think what I really mean is something like the addons on Turbowarp that you can export into a file (Turbowarp's Case of a .json). This is to transfer settings into other computers so you don't have to click on everything again or on school laptops where if you close all tabs, all cookies get cleared. (And for the Communication Error, My Cookies are being cleared, no core settings are being cleared and I am logged in into an account, you can't use guest accounts on these Chromebooks and yes, I am aware that there are only a few settings so yea, pretty small issue but as a convenience factor is i think a good addition)

A (Theoretical) way of adding this in is probably to export a .json file (or .txt) that contains numbers on each line and Beepbox would check each line and turn on/off the according to settings based on which line the setting is for. (For example, Let's say for line 1 on the file, it would be the "Auto Play on Load" setting, if it's 1, turn it on, if it's 0, turn it off)
There is probably another way of doing this but I'm not that good at coding nor know the most optimal methods but (Theoretically) this should work.

**Comment by TheTechRobo** [comment]
Doesn't it already allow you to export as JSON?

**Comment by Micasddsa** [comment]
> Doesn't it already allow you to export as JSON?

No, This isn't for song data, this is for the preferences/settings dropbox.

**Comment by TheTechRobo** [comment]
Ohhh I thought you meant the song settings

**Comment by Mid-The-Modder** [comment]
An option for saving preferences into a file somewhat like a JSON (or, directly as a JSON file for that matter) is a pretty nifty addition, not only for BeepBox, but for other mods using similar preference options. However, no such feature exists in any of them yet.
